src/index.html

```diff
@@ -9,8 +9,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="fonts" href="./fonts/Avenir-Book.ttf">
+    <link rel="fonts" href="./fonts/Avenir-Heavy.ttf">
+    <link rel="fonts" href="./fonts/Avenir.ttc">
+
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=" "
+      method="post"
+      data-qa="big"
+    >
+      <div class="search__bar-big">
+        <label>
+          <input
+            type="text"
+            name="search"
+            class="search__bar-input"
+            placeholder="Try &quot;Los Angeles&quot"
+            data-qa="keypress"
+          >
+        </label>
+      </div>
+    </form>
+    <form
+      action=" "
+      method="post"
+      data-qa="small"
+    >
+      <div class="search__bar-small">
+        <label>
+          <input
+            type="text"
+            name="search"
+            class="search__bar-input-small"
+            placeholder="Try &quot;Los Angeles&quot"
+            data-qa="keypress"
+          >
```

Comment:
follow the checklist
![image](https://user-images.githubusercontent.com/59209337/220041827-2bdbec4f-263c-4cfa-ad51-8266930dc72f.png)

Reply:
fixed